packages/cli-ui/src/components/ProjectManager/ProjectList.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Project } from '../../pages/Projects'
+import ProjectListItem from './ProjectListItem'
+import { useTranslation } from 'react-i18next'
+import css from './style.module.scss'
```

Comment:
When we split imports into logical parts, they read better:
- Library
- App imports
- css

```suggestion
import React from 'react'
import { useTranslation } from 'react-i18next'

import { Project } from '../../pages/Projects'
import ProjectListItem from './ProjectListItem'

import css from './style.module.scss'
```